.backportrc.json

```
@@ -1,7 +1,7 @@
 {
   "repoOwner": "SoftwareAG",
   "repoName": "cumulocity-clients-java",
-  "prTitle": "[BACKPORT][{{targetBranch}}] {{commitMessages}}",
+  "prTitle": "[BACKPORT][{targetBranch}] {commitMessages}",
```

```suggestion
  "prTitle": "[BACKPORT] {commitMessages}",
```
Maybe this version? What do you think?